specification/networkanalytics/resource-manager/Microsoft.NetworkAnalytics/stable/2023-11-15/networkanalytics.json

```diff
@@ -0,0 +1,2571 @@
+      "properties": {
+        "startTimeStamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Sas token start timestamp."
+        },
+        "expiryTimeStamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Sas token expiry timestamp."
+        },
+        "ipAddress": {
+          "type": "string",
+          "description": "Ip Address"
+        }
+      },
+      "required": [
+        "startTimeStamp",
+        "expiryTimeStamp",
+        "ipAddress"
+      ]
+    },
+    "AccountSasToken": {
+      "type": "object",
+      "description": "Details of storage account sas token .",
+      "properties": {
+        "storageAccountSasToken": {
+          "type": "string",
+          "format": "password",
+          "description": "Field to specify storage account sas token.",
+          "x-ms-secret": true
+        }
+      },
+      "required": [
+        "storageAccountSasToken"
+      ]
+    },
+    "Bypass": {
+      "type": "string",
+      "description": "Specifies whether traffic is bypassed for Logging/Metrics/AzureServices. Possible values are any combination of Logging|Metrics|AzureServices (For example, \"Logging, Metrics\"), or None to bypass none of those traffics.",
+      "enum": [
+        "None",
+        "Logging",
+        "Metrics",
+        "AzureServices"
+      ],
+      "x-ms-enum": {
+        "name": "Bypass",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "None",
+            "value": "None",
+            "description": "Represents no bypassing of traffic."
+          },
+          {
+            "name": "Logging",
+            "value": "Logging",
+            "description": "Represents bypassing logging traffic."
+          },
+          {
+            "name": "Metrics",
+            "value": "Metrics",
+            "description": "Represents bypassing metrics traffic."
+          },
+          {
+            "name": "AzureServices",
+            "value": "AzureServices",
+            "description": "Represents bypassing azure services traffic."
+          }
+        ]
+      }
+    },
+    "ConsumptionEndpointsProperties": {
+      "type": "object",
+      "description": "Details of Consumption Properties",
+      "properties": {
+        "ingestionUrl": {
+          "type": "string",
+          "description": "Ingestion url to upload the data.",
+          "readOnly": true
+        },
+        "ingestionResourceId": {
+          "type": "string",
+          "format": "arm-id",
+          "description": "Resource Id of ingestion endpoint.",
+          "readOnly": true,
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.Storage/storageAccounts"
+              }
+            ]
+          }
+        },
+        "fileAccessUrl": {
+          "type": "string",
+          "description": "Url to consume file type.",
```

Comment:
>Url [](http://example.com/codeflow?start=26&length=3)

URL is an acronym and acronyms should be capitalized in the context of human-oriented docs, descriptions, titles.

Reply:
This is cloned from private repo. We will first make required change in the private repo and make a fix in public repo in the next release.